example_yamls/example_msdnet_customdil.yaml

```diff
@@ -1,30 +1,31 @@
 # Example for parameters to excecute
 
 # I/O
-data_tiled_uri: https://tiled-seg.als.lbl.gov/api/v1/metadata/reconstruction/rec20190524_085542_clay_testZMQ_8bit/20190524_085542_clay_testZMQ_
-data_tiled_api_key: <secret>
-mask_tiled_uri: https://tiled-seg.als.lbl.gov/api/v1/metadata/reconstruction/seg-partial-rec20190524_085542_clay_testZMQ_8bit/seg-partial-20190524_085542_clay_testZMQ_
-mask_tiled_api_key: <secret>
-seg_tiled_uri: http://0.0.0.0:8888
-seg_tiled_api_key: <secret>
-
-mask_idx: [10, 201, 222, 493]
-shift: 2
-save_path: results/models
-uid: uid0014
+io_parameters:
+  data_tiled_uri: https://tiled-seg.als.lbl.gov/api/v1/metadata/reconstruction/rec20190524_085542_clay_testZMQ_8bit/20190524_085542_clay_testZMQ_
+  data_tiled_api_key: 
+  mask_tiled_uri: https://tiled-seg.als.lbl.gov/api/v1/metadata/mlex_store/mlex_segm_user/rec20190524_085542_clay_testZMQ_8bit/a558f7b1773855b9453bf6c91079699d
+  mask_tiled_api_key: 
+  seg_tiled_uri: http://0.0.0.0:8888/api/v1/metadata/mlex_store/rec20190524_085542_clay_testZMQ_8bit/results
+  seg_tiled_api_key: 
+  uid: uid0020
 
 model_parameters:
   network: "MSDNet"
   num_classes: 3
   num_epochs: 3
   optimizer: "Adam"
   criterion: "CrossEntropyLoss"
-  weights: null
+  weights: [1, 2, 4]
```

Comment:
Not a string of floats `[0.9, 0.1, 0.0]`?

Reply:
The weights could also take `None` (null in yaml) if we don't want to assign weights for classes, this is just an example. 

I've added an array for the incoming commit.